page-index/page-index.js

```diff
@@ -47,7 +47,7 @@ const pageIndex = [
                 Title: "Installing",
                 type: REACT_PAGE,
                 href: "installation",
-                options: {pg_version: ["11", "10", "9.6"]},
+                options: {pg_version: ["12", "11", "10", "9.6"]},
```

Comment:
Aren't we deprecating 10, and 9.6?

Reply:
We deprecate them, but we still support them and binaries are generated and work. I don't know how the deprecation is actual supposed to work. There is already comment in the text, which mentions deprecation. See on [this page](https://docs.timescale.com/latest/getting-started/installation/windows/installation-windows):

>Note: TimescaleDB requires PostgreSQL 11.4 or 12.0+. Support for PostgreSQL 9.6.3+ and 10.9+ is deprecated and will be removed in a future release.

Reply:
So I think that we should remove `10` and `9.6` for the `v1.7` branch of the docs.  @mfreed opinion?

Reply:
There's no docker of 1.7 with PG10 or 9.6, right?

Reply:
@solugebefola How could the users get binaries for deprecated version in such case? Should we create a special page, which will be a work to do?

Reply:
We have 1.7 binaries for PG10 and 9.6?

Reply:
Regardless, we should merge this now so folks have a way to access the PG12 binary and remove the others in a separate PR if necessary.

Reply:
@solugebefola I built and published docker images for 9.6 and 10.
1.7 binaries for 9.6 and 10 are already available for download through our docs. Do you suggest to remove binaries, which are already available for users.

Reply:
I suggest to keep access to 9.6 and 10 even they are replicated and remove them when we remove support, i.e., from 2.0.